packages/remix/README.md

```diff
@@ -67,9 +67,9 @@ Whereas the environment variables reflect the respective configuration of the ap
 The SDK ships with a convenient function that mounts the authentication routes: sign-in, sign-in callback and the sign-out route:
 
 ```ts
-// app/routes/api/logto/$action.ts
+// app/routes/api.logto.$action.ts
```

Comment:
Why do we change the slashes to dots?

Reply:
Because that's required by remix v2 afaik

Reply:
I couldn't get it working without doing this

Reply:
Ah yes seems it's a new thing from remix v2 and is called out here: https://remix.run/docs/en/main/start/v2#file-system-route-convention

Reply:
OK, np